bindata/v4.1.0/kube-controller-manager/cluster-policy-controller-role.yaml

```diff
@@ -0,0 +1,13 @@
+# This role is necessary for cluster-policy-controller to identify its own pod and owner for emitting events
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: openshift-kube-controller-manager
+  name: system:openshift:cluster-policy-controller
```

Comment:
is it okay to add a new role/rolebinding?

Reply:
we will get the following warning unless we add the pod rule

```
W0708 12:14:57.795809       1 builder.go:209] unable to get owner reference (falling back to namespace): pods "kube-controller-manager-ip-10-0-168-17.eu-central-1.compute.internal" is forbidden: User "system:kube-controller-manager" cannot get resource "pods" in API group "" in the namespace "openshift-kube-controller-manager" 
```

The events will be emitted by the node (owner of the pod)

Reply:
removing, will be resolved by passing the owner reference directly